pyvsystems/wrapper.py

```diff
@@ -24,9 +24,17 @@ def request(self, api, post_data=''):
                     return requests.post(url, data=post_data, headers=headers).json()
             else:
                 if self.timeout:
-                    return requests.get(url, headers=headers, timeout=self.timeout).json()
+                    req = requests.get(url, headers=headers, timeout=self.timeout).json()
```

Comment:
`req` should change to `resp` (response). `req` always defines as "request" variable.